packages/main/src/themes/ValueStateMessage.css

```diff
@@ -87,6 +90,55 @@
 }
 
 .ui5-valuestatemessage-popover::part(header),
-.ui5-suggestions-popover-with-value-state-header::part(header) {
+.ui5-popover-with-value-state-header::part(header),
+.ui5-popover-with-value-state-header-phone::part(header) {
 	margin-bottom: 0;
 }
+
+.ui5-popover-with-value-state-header .ui5-valuestatemessage-root {
+	padding: var(--_ui5_value_state_message_padding);
+}
+
+.ui5-popover-with-value-state-header-phone .ui5-valuestatemessage-root {
+	padding: var(--_ui5_value_state_message_padding_phone);
+	width: 100%;
+}
+
+.ui5-popover-with-value-state-header-phone .ui5-input-value-state-message-icon {
+	left: var(--_ui5_value_state_message_icon_offset_phone);
+}
+
+.ui5-popover-with-value-state-header-phone .ui5-valuestatemessage-header{
+	position: relative;
+	flex: none;
+	top: 0;
+	left: 0;
+}
+
+.ui5-popover-with-value-state-header-phone::part(content) {
+	padding: unset;
+	overflow: hidden;
+	display: flex;
+	flex-direction: column;
+}
+
+.ui5-popover-with-value-state-header-phone .ui5-valuestatemessage-root + ui5-list {
```

Comment:
Yes, because the scroll should now only affect the ui5-list. So far, on mobile devices, the value state message was positioned in the sticky dialog header. The dialog content was scrollable. I have moved the value state message in the dialog content. Value state message should not be scrolled along with the content, but stay on top. So this style applies to the ui5-list and makes it scrollable instead of the whole content.